config/postcreate-hooks/kubestellar.yaml

```diff
@@ -204,6 +206,8 @@ spec:
               - "0.24.0-experiment.5"
               - --set
               - "wds_cp_name={{.ControlPlaneName}}"
+              - --set
+              - "verbosity={{.TransportControllerVerbosity}}"
```

Comment:
Not introduced by this PR. I see this value is defaulted to 4 by ocm-transport-controller-chart/values.yaml. Should it be a lower level for a user?

Reply:
Not sure which layer you are questioning.

The kubestellar PostCreateHook is, as far as I can tell (https://github.com/kubestellar/kubestellar/blob/main/docs/content/direct/packaging.md#kubestellar), only used in our E2E test setup.

The transport controller Helm chart is called by the core chart, which has its own defaults. The transport-controller Helm chart is not documented for use by users (yet?). Yes, we could change the default in a different PR.

Reply:
I was asking about the latter. Thanks for the explanations!